model/src/main/proto/platform_parameter.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// It will correspond to a single Platform Parameter Value.
+message PlatformParameter {
+    // Value of a parameter can be of any one of the following type.
+    oneof value {
+        // Boolean type parameter value.
+        bool boolean = 2;
+        // Integer type parameter value.
+        int32 integer = 3;
+        // String type parameter value.
+        string string = 4;
+    }
+}
+
+// Storage format of platform parameters while the app is running.
```

Comment:
Do we actually need this class if it's only for when the app is running? Why not use a Kotlin map in that case?

Reply:
Yeah a normal Map aould work too. /cc @vinitamurthi

Reply:
Not an issue now as We will replace it with a normal Kotlin Map

Reply:
Got it. I think this also helps address the way to store it, too

Reply:
Yeah